Depends-On: 0001_win32_resource_template.patch
Touch-Files: src/win32/installer/build_installer.py, src/win32/installer/installer_oss_64bit.wxs
Scope: installer
diff --git a/src/win32/installer/build_installer.py b/src/win32/installer/build_installer.py
index 4bccad8c5..8b4270cc7 100644
--- a/src/win32/installer/build_installer.py
+++ b/src/win32/installer/build_installer.py
@@ -131,7 +131,7 @@ def run_wix4(args) -> None:
         f'Software\\Google\\Update\\ClientState\\{{{omaha_guid}}}'
     )
   elif branding == 'Mozc':
-    upgrade_code = 'DD94B570-B5E2-4100-9D42-61930C611D8A'
+    upgrade_code = '1157b660-5033-4cc4-9907-7d26ba1ada5c'
 
   omaha_channel_type = 'dev' if version.IsDevChannel() else 'stable'
   vs_configuration_name = 'Debug' if args.debug_build else 'Release'
diff --git a/src/win32/installer/installer_oss_64bit.wxs b/src/win32/installer/installer_oss_64bit.wxs
index b3b308e60..c27a068c9 100644
--- a/src/win32/installer/installer_oss_64bit.wxs
+++ b/src/win32/installer/installer_oss_64bit.wxs
@@ -57,8 +57,8 @@
     http://msdn.microsoft.com/en-us/library/aa367850.aspx.
     So we change the product ID for every new version.
   -->
-  <Package Name="Mozc" Language="1041" Codepage="932" Version="$(var.MozcVersion)" Manufacturer="Google LLC" UpgradeCode="$(var.UpgradeCode)" InstallerVersion="500">
-    <SummaryInformation Keywords="Installer" Description="Mozc インストーラー" Manufacturer="Google LLC" Codepage="932" />
+  <Package Name="Mozc4med" Language="1041" Codepage="932" Version="$(var.MozcVersion)" Manufacturer="Mozc Project, MATANUKI" UpgradeCode="$(var.UpgradeCode)" InstallerVersion="500">
+    <SummaryInformation Keywords="Installer" Description="Mozc4med インストーラー" Manufacturer="Mozc Project, MATANUKI" Codepage="932" />
 
     <!--
        While obtaining the OS build number from the registry does not seem to be officially
@@ -101,11 +101,11 @@
     <Property Id="OS_BUILD_NUMBER" Secure="yes">
       <RegistrySearch Id="CurrentBuildNumberValue" Type="raw" Root="HKLM" Key="SOFTWARE\Microsoft\Windows NT\CurrentVersion" Name="CurrentBuildNumber" />
     </Property>
-    <Launch Condition="(OS_BUILD_NUMBER AND (OS_BUILD_NUMBER &gt;= 17763)) OR (REMOVE=&quot;ALL&quot;)" Message="Mozc をインストールするには Windows 10 1809 以降にアップグレードする必要があります。" />
+    <Launch Condition="(OS_BUILD_NUMBER AND (OS_BUILD_NUMBER &gt;= 17763)) OR (REMOVE=&quot;ALL&quot;)" Message="Mozc4med をインストールするには Windows 10 1809 以降にアップグレードする必要があります。" />
 
     <!-- Check if the user have the administrative privileges. -->
-    <Launch Condition="Privileged" Message="Mozc をインストールするには管理者権限が必要です。" />
-    <Media Id="1" Cabinet="Mozc.cab" EmbedCab="yes" CompressionLevel="high" />
+    <Launch Condition="Privileged" Message="Mozc4med をインストールするには管理者権限が必要です。" />
+    <Media Id="1" Cabinet="Mozc4med.cab" EmbedCab="yes" CompressionLevel="high" />
 
     <?if $(var.MozcTIP64ArmPath) == "" and $(var.MozcTIP64XPath) == "" and $(var.MozcUniversalInstaller) == "" ?>
       <!--
@@ -136,10 +136,10 @@
     </Upgrade>
 
     <UI>
-      <Error Id="4000" Message="新しいバージョンの Mozc が既にインストールされています。" />
+      <Error Id="4000" Message="新しいバージョンの Mozc4med が既にインストールされています。" />
     </UI>
 
-    <Feature Id="MozcInstall" Title="Mozc" Level="1">
+    <Feature Id="MozcInstall" Title="Mozc4med" Level="1">
       <ComponentRef Id="MozcTIP32" />
       <ComponentRef Id="MozcTIP64" />
       <?if $(var.MozcTIP64ArmPath) != "" and $(var.MozcTIP64XPath) != "" ?>
@@ -241,7 +241,7 @@
     </InstallExecuteSequence>
 
     <StandardDirectory Id="ProgramFilesFolder">
-      <Directory Id="MozcDir" Name="Mozc">
+      <Directory Id="MozcDir" Name="Mozc4med">
         <Component Id="MozcConverter">
           <File Id="mozc_server.exe" Name="mozc_server.exe" DiskId="1" Checksum="yes" Source="$(var.MozcServer64Path)" Vital="yes" />
         </Component>
@@ -379,7 +379,7 @@
       mozc_renderer.exe automatically after user logon.
     -->
     <Component Id="PrelaunchProcesses" Directory="TARGETDIR">
-      <RegistryValue Id="RunBroker" Root="HKLM" Key="Software\Microsoft\Windows\CurrentVersion\Run" Name="Mozc Prelauncher" Action="write" Type="string" Value="&quot;[MozcDir]mozc_broker.exe&quot; --mode=prelaunch_processes" />
+      <RegistryValue Id="RunBroker" Root="HKLM" Key="Software\Microsoft\Windows\CurrentVersion\Run" Name="Mozc4med Prelauncher" Action="write" Type="string" Value="&quot;[MozcDir]mozc_broker.exe&quot; --mode=prelaunch_processes" />
     </Component>
   </Package>
 </Wix>
